pandas/tests/test_index.py

```diff
@@ -418,7 +418,7 @@ def test_is_(self):
 
```

Comment:
need a test for passing invalid ``method`` (put in Base so its tested for all Index classes). Though since you override the checking, might be necessary to have some individual tests for this.